CONTRIBUTING.md

```diff
@@ -47,12 +47,13 @@ Creating a pull request is fairly simple, just make sure it focuses on a single
 
 We would greatly appreciate the code submitted to be of a consistent style with other code in disnake. This project follows PEP-8 guidelines (mostly) with a column limit of 100 characters.
 
-Run these commands to install [`nox`](https://nox.thea.codes/) and [`pdm`](https://pdm.fming.dev/), as well as the required dependencies in your environment, and to set up [`pre-commit`](https://pre-commit.com/#quick-start) hooks.
+Run these commands to install [`PDM`](https://pdm.fming.dev/), as well as the required dependencies in your environment, and to set up [`pre-commit`](https://pre-commit.com/#quick-start) hooks.
```

Comment:
```suggestion
Run these commands to install [`PDM`](https://pdm.fming.dev/) as well as the required dependencies in your environment, and to set up [`pre-commit`](https://pre-commit.com/#quick-start) hooks.
```

Reply:
offloaded installation responsibility to the official pdm docs in https://github.com/DisnakeDev/disnake/pull/953/commits/3fb25c514614fb92b9257165e9c1aa973b886c20, so this suggestion is moot and superseded.